### ISSUE TITLE
Fix breaking changes in FastBoot 1.0

### ISSUE THREAD

**Issue by simonihmig** [RSSchermer/ember-spectrum-color-picker#18]
The current `ember-cli-fastboot` releases (1.0.0-rc.1 and above) introduce breaking changes. These will most likely break your current FastBoot implementation.

See https://github.com/ember-fastboot/ember-cli-fastboot/issues/387 for more information and a guide on how to fix your addon. Also you may want to visit the `-fastboot` Slack channel to get help from other users.

*Note: this issue has been created automatically, by searching for certain patterns in your code. If you think this has been falsely created, feel free to close!*

**Comment by bpcarlsen** [comment]
This update is preventing the css file from loading. It's correctly being included in the vendor.css, but you are wrapping the the contents of the css in the typeof condition as well when it should only be the js file.

**Comment by RSSchermer** [comment]
I push a possible fix to `fastboot-fix`. Could you swap out your `ember-spectrum-color-picker` dependency for:

```
"ember-spectrum-color-picker": "rsschermer/ember-spectrum-color-picker#fastboot-fix"
```

And check whether that resolves the issue for you? If it does I'll merge it into master and publish to npm.

**Comment by bpcarlsen** [comment]
Yes that fixed it.

**Comment by RSSchermer** [comment]
Merged and published as 0.7.2.
